### PR TITLE
Publish completion status on successfull completion of scorm course.

### PR DESCRIPTION
**Pull request by tehreem-sadat**
Ticket LInk: https://edlyio.atlassian.net/browse/EDE-1081

### CODE REVIEW

File: openedxscorm/scormxblock.py [hunk 1]
```diff
@@ -331,6 +335,35 @@ def publish_grade(self):
...
+        """
+        if not completion_waffle.waffle().is_enabled(completion_waffle.ENABLE_COMPLETION_TRACKING):
+            return
+
+        if XBlockCompletionMode.get_mode(self) != XBlockCompletionMode.COMPLETABLE:
+            return
+
+        completion_value = 0.0
+        if not self.has_score:
+            # componenet does not have any score
```
Comment (mumarkhan999) [inline openedxscorm/scormxblock.py]:
```suggestion
            # component does not have any score
```

File: openedxscorm/scormxblock.py [hunk 2]
```diff
@@ -331,6 +335,35 @@ def publish_grade(self):
...
+            if self.get_completion_status() == 'completed':
+                completion_value = 1.0
+        else:
+            if self.get_completion_status() in ['passed', 'failed']:
+                completion_value = 1.0
+
+        data = {
+            'completion': completion_value
+        }
+        self.runtime.publish(self, "completion", data)
```
Comment (mumarkhan999) [inline openedxscorm/scormxblock.py:365]:
```suggestion
        self.runtime.publish(self, 'completion', data)
```

**Review by mumarkhan999 — APPROVED** [review]

File: openedxscorm/scormxblock.py (hunk 1 above)
Comment (tehreem-sadat) [reply openedxscorm/scormxblock.py]: done.

File: openedxscorm/scormxblock.py (hunk 2 above)
Comment (tehreem-sadat) [reply openedxscorm/scormxblock.py:365]: for consistency, I am using double quotes.

**Pull request merged**